ci-operator/config/openshift/installer/master.json

```diff
@@ -41,7 +41,7 @@
     {
       "as": "unit",
       "from": "src",
-      "commands": "go test ./installer/pkg/..."
+      "commands": "go test ./pkg/... ./installer/pkg/..."
```

Comment:
Why not `./...` ?

Reply:
> Why not `./...` ?

We are currently leaning on Go's test framework for [smoke tests][1].  We may drop those soon in favor of more e2e-aws coverage (#1271 is groundwork for this expansion), but until then we need to keep this job out of `tests/smoke`.  A white-list like this is one way to do that.  There may also be blacklist approaches, but I think we'd need to include `find` in our invocation if we wanted to do that.

[1]: https://github.com/openshift/installer/tree/245fc0f6313697ecbb2732c4f7184aebc8f0be52/tests/smoke